pkg/wallet/api.go

```diff
@@ -90,7 +90,7 @@ type Pack interface {
 type Envelope struct {
 	Message    []byte
 	FromVerKey string
-	ToVerKeys  string
+	ToVerKeys  []string
```

Comment:
Are these encoded keys?

(also wondering why string is being used for keys, instead of using []byte)

Reply:
they are encoded keys with base58

Reply:
Issue #272 again

Reply:
I think all public keys should encoded with base58